aioftp/server.py

```diff
@@ -1133,6 +1133,8 @@ def _build_mlsx_facts_from_stats(self, stats):
         }
 
     async def build_mlsx_string(self, connection, path):
+        if not await connection.path_io.exists(path):
+            return ""
         stats = await connection.path_io.stat(path)
         facts = self._build_mlsx_facts_from_stats(stats)
```

Comment:
I think empty spec is a bad idea. Client will get empty line and it is pretty confused. At least Type=unknown and filename will be in spec line with such changes
```suggestion
        if not await connection.path_io.exists(path):
            facts = {}
        else:
            stats = await connection.path_io.stat(path)
            facts = self._build_mlsx_facts_from_stats(stats)
```